packages/migrate/src/__tests__/DbPull.test.ts

```diff
@@ -21,6 +21,25 @@ const ctx = jestContext.new().add(jestConsoleContext()).add(jestProcessContext()
 // To avoid the loading spinner locally
 process.env.CI = 'true'
 
+// describeIf is making eslint not happy about the names
+/* eslint-disable jest/no-identical-title */
+
+// We want to remove unique IDs to have stable snapshots
+// Example:
+// `PK__User__3213E83E450CDF1F` will be changed to `PK__User__RANDOM_ID_SANITIZED`
```

Comment:
The way we deal with that in engines tests is to make the names explicit in our SQL migrations. I've had many bad experiences with test setups that post-process the output.